### PR TITLE
Opscode -> Chef and rdoc -> md

### PR DESCRIPTION
**Pull request by tas50**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-Mixlib::Authentication provides a class-based header signing authentication object, like the one used in Chef. 
+Mixlib::Authentication provides a class-based header signing authentication object, like the one used in Chef.
 
-== License
-
-Author:: Christopher Brown (<cb@opscode.com>)
-Copyright:: Copyright (c) 2009 Opscode, Inc.
-License:: Apache License, Version 2.0
+## License
+- Author:: Christopher Brown ([cb@chef.io](mailto:cb@chef.io))
```
Comment (lamont-granquist) [inline README.md]: that e-mail address won't be terribly useful.

**Comment by lamont-granquist** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (tas50) [inline README.md]: Just as bad as the last one ;)

Comment (lamont-granquist) [inline README.md]: Yeah but you went through the trouble of making a broken e-mail address clickable...  lol...

Comment (tas50) [inline README.md]: That's just Atom formatting the markdown automatically. No trouble on my part

Comment (lamont-granquist) [inline README.md]: ah, computers being helpful...

**Pull request merged**